x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -82,7 +82,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
         const error = await find.byCssSelector('.euiFormErrorText');
         expect(await error.getVisibleText()).equal(
-          'The length of the title is too long. The maximum length is 160.'
+          'The length of the title is too long. The maximum length is 160 characters.'
```

Comment:
How about a similar test in this file in the comment kinda like `'adds a comment to a case'` but with a comment that is too long?

Reply:
I have intentionally not put description and comment validations in e2e. Because it takes lot of time in execution. It is typing all 30K characters in markdown 😄